src/handwritten/task.py

```diff
@@ -1220,16 +1223,18 @@ class _TaskAlternateConstructor(Task):
     # Setting __slots__ avoids TypeError: __class__ assignment: 'Base' object layout differs from 'Derived'.
     __slots__ = []
 
-    def __init__(self, task_handle, interpreter):
+    def __init__(self, task_handle, interpreter, close_on_exit=False):
```

Comment:
Please make this parameter required to ensure we don't omit it by accident. All calls to `_TaskAlternateConstructor` correctly specify it.

```
PS C:\dev\nidaqmx-python> rg "_TaskAlternateConstructor"
src\handwritten\task.py
1217:class _TaskAlternateConstructor(Task):

src\handwritten\system\storage\persisted_task.py
147:        return task._TaskAlternateConstructor(task_handle, self._interpreter, close_on_exit)

generated\nidaqmx\task.py
1217:class _TaskAlternateConstructor(Task):

generated\nidaqmx\system\storage\persisted_task.py
147:        return task._TaskAlternateConstructor(task_handle, self._interpreter, close_on_exit)
```

```suggestion
    def __init__(self, task_handle, interpreter, close_on_exit):
```